tests/fast-integration/skr-svcat-migration-test/deploy-sample-resources.js

```diff
@@ -252,7 +249,6 @@ async function deploy() {
   await waitForPodWithLabel('app', 'service-broker-proxy-k8s', 'kyma-system');
 
   try {
-    await waitForClusterServiceBroker('helm-broker', 5 * 60 * 1000);
```

Comment:
nit: technically this whole test can be removed after svcat is removed from kyma but I think we can start iteratively and disable it by parts